composer.json

```diff
@@ -15,7 +15,7 @@
     ],
     "require": {
         "php": ">=5.6",
-        "guzzlehttp/guzzle": "^6.3"
+        "guzzlehttp/guzzle": "^5"
```

Comment:
The version in the core is v3 the last time I've checked

Reply:
Nope, its v5: https://github.com/PrestaShop/PrestaShop/blob/develop/composer.json#L26
```
"guzzlehttp/guzzle": "~5.0"
```

Reply:
it's a good news :+1: